src/allowed_args.cpp

```diff
@@ -311,10 +312,19 @@ static void addGeneralOptions(AllowedArgs &allowedArgs, HelpMessageMode mode)
                     DEFAULT_PERSIST_MEMPOOL))
         .addArg("prune=<n>", requiredInt,
             strprintf(_("Reduce storage requirements by pruning (deleting) old blocks. This mode is incompatible with "
-                        "-txindex and -rescan. "
+                        "-txindex, -rescan, and -prunewithmask"
                         "Warning: Reverting this setting requires re-downloading the entire blockchain. "
                         "(default: 0 = disable pruning blocks, >%u = target size in MiB to use for block files)"),
                     MIN_DISK_SPACE_FOR_BLOCK_FILES / 1024 / 1024))
+        .addArg("prunewithmask={true,false,0,1}", optionalBool,
+            strprintf(_("Reduce storage requirements by pruning (deleting) old blocks. This mode is incompatible with "
+                        "-txindex, -rescan, and -prune "
+                        "Warning: Reverting this setting requires re-downloading the entire blockchain. "
+                        "(default: 0 = disable pruning blocks, 1 = prune blocks using a randomly generated mask)")))
+        .addArg("prunethreshold=<n>", requiredInt,
+            strprintf(_("A normalized threshold to determine how"
+                        "many blocks to keep when prunning with hashmask (default: %u)"),
+                    DEFAULT_THRESHOLD_PERCENT))
```

Comment:
If you have this as a tweak, you don't need to .addArg.  All tweaks are automatically added as args.

Reply:
removed this entry from args list.